### DISCUSSION TITLE
Notifications needs a way to filter "closed" or "completed" items

### DISCUSSION THREAD

**Discussion by NSExceptional** [community/community#78187] [Mobile]
### Select Topic Area

Product Feedback

### Body

Working at a large organization that makes dozens of PRs a day with my team assigned, the notifications tab quickly becomes unwieldy. I would like a way to either a) mark all completed items as read/done, or b) filter the list to only show those items so I can manually mark them as read

**Comment by iskae** [comment]
Hi @NSExceptional 👋 

Thank you for reaching out ❤️ 

May I ask which platform are you using? We are offering multi-selecting capabilities via long tapping on any notification items and also offering `Select/Unselect all` through the apps top bar. You can also configure your notification actions via notification settings on your profile and mark notifications as done/undone or read/unread. On top of these we are also offering filtering capabilities in notifications screen too, so that you can filter by repository/state or read/unread state too 😊 

Was this the thing that you were looking for or were you referring to adding filtering capabilities for filtering out closed issues/prs/discussions?

**Reply by NSExceptional** [reply]
Hey! Allow me to clarify. What I mean is, I want a way to filter on the status of items. Firstly, I know there are all types of content in notifications. PRs, discussions, issues, etc. And because of this, filtering options are limited:

<img width=30% src=https://github.com/community/community/assets/8371943/a6b3ee5a-855d-4891-9e9e-c54a2b4b132c>

But, most of these types of content can be broken into two additional categories:

- Open: open issues, open PRs, and unanswered discussions
- Closed: closed/completed issues, closed/merged PRs, and answered discussions

Everything else kinda falls into an obscure third category I suppose, but I am not concerned with those here. I want a way to filter the notifications table to only show one of the two categories above: all open issues/PRs/etc, or all closed/merged issues/PRs/etc. And I want this because I want to be able to quickly "clear" out anything that no longer needs my attention (merged PRs, closed issues, etc).

**Reply by iskae** [reply]
Oooh thanks so much for the explanation ❤️ 

I think this makes sense 👍 Let me create an internal tracking issue for this and let us discuss this further 🙇 

Please keep the feature requests or bug reports coming in ❤️ 

Cheers!
